pkg/fixer/fixes/useassignmentoperator.go

```diff
@@ -0,0 +1,41 @@
+package fixes
+
+import (
+	"bytes"
+)
+
+type UseAssignmentOperator struct{}
+
+func (*UseAssignmentOperator) Key() string {
+	return "use-assignment-operator"
+}
+
+func (*UseAssignmentOperator) WholeFile() bool {
+	return false
+}
+
+func (*UseAssignmentOperator) Fix(in []byte, opts *RuntimeOptions) (bool, []byte, error) {
+	lines := bytes.Split(in, []byte("\n"))
+
+	// this fix must have locations
+	if len(opts.Locations) == 0 {
+		return false, nil, nil
+	}
+
+	for _, loc := range opts.Locations {
+		if loc.Row > len(lines) {
+			return false, nil, nil
+		}
+
+		if loc.Col != 1 {
+			// current impl only understands the first column
```

Comment:
OK, there's probably something I don't understand here 😅

Reply:
So I think for this one I am correct in that it shows as char one: 

```
regal $ cat test.rego
package wow

allow = true #thjings
regal $ cat test.rego
package wow
allow = true
regal $ regal lint test.rego --disable-all --enable=use-assignment-operator
Rule:           use-assignment-operator                                                 
Description:    Prefer := over = for assignment                                         
Category:       style                                                                   
Location:       test.rego:2:1                                                           
Text:           allow = true                                                            
Documentation:  https://docs.styra.com/regal/rules/style/use-assignment-operator        

1 file linted. 1 violation found.

```

Reply:
Ah, right, since the `=` is nowhere to be found in the AST. I'll take a stab at fixing this in the rule instead, as that'll get us more accurate reporting in the CLI output as well.

Reply:
I can change the rule with that in mind.

Reply:
ca8e0a2e6b2130299667c2123fee7016351fff51 makes this change